### DISCUSSION TITLE
[Wiki/Blog] Use NvChad as a JavaScript/Go/Rust IDE

### DISCUSSION THREAD

**Discussion by jellydn** [NvChad/NvChad#1580] [Ideas]
It would be great and bring better DX for new users using NvChad by adding some sections on wiki or blog posts such as Use NvChad as a JavaScript/Go/Rust IDE. For example, this is `Use Vim as a JavaScript IDE
` post from SpaceVim  https://spacevim.org/use-vim-as-a-javascript-ide/

**Comment by siduck** [comment]
@jellydn good idea, we use docusaurus to generate our statiic site, so you can check the blog feature https://docusaurus.io/blog

**Reply by jellydn** [reply]
Cool. That sounds good. Then the homepage or FAQ sections could have links to blog posts.
